Source/UI/NewUI/PaymentForm/View/ExpiryDateView.swift

```diff
@@ -146,6 +146,9 @@ public final class ExpiryDateView: UIView {
       Constants.LocalizationKeys.PaymentForm.ExpiryDate.Error.past :
       Constants.LocalizationKeys.PaymentForm.ExpiryDate.Error.invalid
     style?.error?.isHidden = isHidden
+    if !isHidden {
+      delegate?.update(expiryDate: nil)
+    }
```

Comment:
I tried finding but I'm unable, could you please show me where the `ExpiryDateView` is calling delegate with non nil  values? This code shows some logic problems in my understanding so far 🤔